docs/runtimes/image-build.md

```diff
@@ -0,0 +1,192 @@
+(build-function-image)=
+# Build function image
+
+As discussed in {ref}`images-usage`, MLRun provides pre-built images which contain the components necessary to execute
+an MLRun runtime. In some cases, however, custom images need to be created. 
+This page details this process and the available options.
+
+## When is a build required?
+
+In many cases an MLRun runtime can be executed without having to build an image. This will be true when
```

Comment:
In many cases an MLRun runtime can be executed without having to build an image. This is true when
the basic MLRun images fulfill all the requirements for the code to execute. It is required to build an image 
if one of the following is true: